serverless/main.mk

```diff
@@ -19,6 +19,9 @@ CMD_SLS_SERVICE_INVOKE = $(SLS) invoke --function $(SVC) --path $(EVENT_FILE) --
 CMD_SLS_SERVICE_DESTROY = $(SLS) remove --config $(SLS_FILE) --service $(SVC) --verbose --region $(AWS_REGION) --env $(ENV) --profile $(AWS_PROFILE)
 CMD_SLS_SERVICE_BUILD = cd $(ROOT_DIR)/$(PROJECT_PATH) && make
 CMD_SLS_SERVICE_SECRETS = $(CMD_SERVICE_SECRETS_PUSH)
+# This works with "serverless-domain-manager" plugin and provide domain creation and remove
+CMD_SLS_SERVICE_DOMAIN_CREATE = $(SLS) create_domain --verbose --region $(AWS_REGION) --env $(ENV) --profile $(AWS_PROFILE)
```

Comment:
Looks good, but I'm wondering hwy cmd is `..._DOMAIN_CREATE` and sls command is `create_domain` (order). Maybe we should match the wording based on the sls original command.

Reply:
I thought that I should use syntaxis as that was before me - and it's like: 
CMD(command)_SLS_SERVICE(canvas)_BUILD(what to do). If you insist - I can change.